metrics_utility/management/commands/gather_automation_controller_billing_data.py

```diff
@@ -109,8 +110,22 @@ def _handle_datelike(self, value, help=''):
         if not value:
             return None
 
-        if value.isdigit():
-            raise UnparsableParameter(f'Bare numbers are not valid ({help})')
+        if (
+            isinstance(value, str)
+            and (not value.endswith('m') and not value.endswith('d'))
+            and not isinstance(value, datetime.datetime)
```

Comment:
if you're getting a datetime here, somethings is getting called twice, or the tests are passing in datetimes instead of strings (and thus not testing real values)